app/views/admin/questions/_level.html.slim

```diff
@@ -0,0 +1,12 @@
+.progress
+  - l = @question.level
+  - w = '100%'
+  - c = "danger"
+  - if l == 'beginner'
+    - w = '33%'
+    - c = "success"
+  - if l == 'good'
+    - w = '66%'
+    - c = "warning"
+  .progress-bar class = "progress-bar-#{c}" aria-valuemax="100" role="progressbar" style="width:#{w}"
+    = l
```

Comment:
Obviously this code can not be in the view.
Does it mean that I must store this code in another place and add here a link to it?